packages/e2e-test-utils-playwright/src/request-utils/menus.ts

```diff
@@ -0,0 +1,108 @@
+/**
+ * Internal dependencies
+ */
+import type { RequestUtils } from './index';
+
+export interface MenuData {
+	title: string;
+	content: string;
+}
+export interface Menu {
+	id: number;
+	content: string;
+	status: 'publish' | 'future' | 'draft' | 'pending' | 'private';
+}
+
+/**
+ * Create a classic menu
+ *
+ * @param {string} name Menu name.
+ * @return {string} Menu content.
+ */
+export async function createClassicMenu( this: RequestUtils, name: string ) {
+	const menuItems = [
+		{
+			title: 'Home',
```

Comment:
This is quite a generic name. Is it possible that the test could pass but because it found a different navigation with a "Home" option? Maybe its safer to use a less generic name.

Reply:
Makes sense. This function was way more complicated on the puppeteer tests, so I made it much simpler for this test, and maybe we can refactor it later if it's useful for other tests too.

Reply:
FYI I've found this to be a problem in the past. I agree with Ben it should use a page name that is unusual and quite long.

Reply:
I changed it to "Custom link", shall I make it longer?

Reply:
That should be good enough 👍